devops/actions/llvm_test_suite_win/action.yml

```diff
@@ -0,0 +1,87 @@
+name: 'Run llvm-test-suite testing'
+description: 'Run E2E SYCL tests from llvm-test-suite'
+inputs:
+  test_ref:
+    description: 'commit-ish identifier for test repo'
+    required: true
+    default: 'intel'
+  sycl_artifact:
+    description: 'Name of the artifact, that contains compiler toolchain'
+    required: true
+  lit_artifact:
+    description: 'Name of the artifact, that contains LIT tool'
+    required: true
```

Comment:
Do we need that? IIRC, I've switched the Linux action to use the LIT from the repo checkout instead.